packages/documentation/src/pages/architecture/unit-tests.mdx

```diff
@@ -0,0 +1,120 @@
+## How do unit tests work?
+
+#### Where they live
+- Tests can live anywhere in the `src` directory, but in general are located in a directory close to the code or application that they are written to test.
+- The `.unit.spec.js` file extension is used for unit tests, while `.e2e.spec.js` is used for end-to-end (E2E) tests.
```

Comment:
may indent this under the line above and break it into two lines- bold important words so this is easier to scan.

> **Unit tests:**  Mocha will run every test in the `/src` directory with the `.unit.spec.js` extension
> **End to end tests:** Jest will run every test  in the `/src` directory with the `.e2e.spec.js` extension